mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/ComputedPropertyDefinition.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.internal.mapper.processor.entity;
+
+import com.datastax.oss.driver.internal.mapper.processor.util.generation.PropertyType;
+import com.squareup.javapoet.CodeBlock;
+import java.util.Optional;
+
+public class ComputedPropertyDefinition implements PropertyDefinition {
+
+  private final CodeBlock cqlName;
+  private final CodeBlock cqlResultName;
+  private final String getterName;
+  private final String setterName;
+  private final PropertyType type;
+
+  public ComputedPropertyDefinition(
+      String javaName,
+      Optional<String> customCqlName,
+      String formula,
+      String getterName,
+      String setterName,
+      PropertyType type,
+      CqlNameGenerator cqlNameGenerator) {
+    this.cqlResultName =
+        customCqlName
+            .map(n -> CodeBlock.of("$S", n))
+            .orElse(cqlNameGenerator.buildCqlName(javaName));
+    // the queried name is "formula as alias"
+    this.cqlName =
+        CodeBlock.of("$S + $S +", formula, " as ").toBuilder().add(this.cqlResultName).build();
```

Comment:
Is there a better way to do this?  The generated code looks like:

```java
    return selectFrom
        .column("id")
        .column("c_id")
        .column("v")
        .raw("writetime(v)" + " as " +"writetime")
        .raw("ttl(v)" + " as " +"myttl");
```

Which works, but would have been nice to do it as one clean string constant.

Reply:
You can alias raw expressions:
```java
.raw("writetime(v)").as("writetime")
.raw("ttl(v)").as("myttl");
```

Reply:
Nice, that's definitely better.  It will require some reorienting of `ComputedPropertyDefinition` and how it is used in `EntityHelperSelectStartMethodGenerator`, but I should probably do that anyways.

Reply:
Was a little easier to do that than i originally anticipated (6b24969)